README.md

```diff
@@ -100,7 +100,20 @@ Second, the plugin looks for the `CASC_JENKINS_CONFIG` environment variable. The
 - A full path to a single file. For example, `/var/jenkins_home/casc_configs/jenkins.yaml`.
 - A URL pointing to a file served on the web. For example, `https://acme.org/jenkins.yaml`.
 
-If `CASC_JENKINS_CONFIG` points to a folder, the plugin will recursively traverse the folder to find file (suffix with .yml,.yaml,.YAML,.YML), but doesn't contain hidden files or hidden subdirectories. It follows symbolic links for both files and directories.
+If `CASC_JENKINS_CONFIG` points to a folder, the plugin will recursively traverse the folder to find file(s) with .yml,.yaml,.YAML,.YML suffix. It will exclude hidden files or files that contain a hidden folder in **any part** of the full path. It follows symbolic links for both files and directories.
+
+#### Exclusion examples:
```

Comment:
this is too detailed and eye grabbing for the front page imo,

possibly ok if it would be hidden behind and expandable section with summary / details,

(especially the red crosses are very eye grabbing and distracting)

Reply:
True, did not consider that.

Reply:
Something like this?
![image](https://user-images.githubusercontent.com/14072681/83361991-c3d95700-a385-11ea-8cc8-69f93bd3e9f2.png)

Reply:
Exactly :)

Reply:
@timja I have put it into collapsible section now
collapsed
![image](https://user-images.githubusercontent.com/14072681/83428195-dc02b200-a429-11ea-93b3-6c1a81bc400a.png)
expanded
![image](https://user-images.githubusercontent.com/14072681/83428256-f50b6300-a429-11ea-8a25-c465e1385910.png)